beacon-chain/p2p/subnets.go

```diff
@@ -156,3 +247,11 @@ func (s *Service) subnetLocker(i uint64) *sync.RWMutex {
 	}
 	return l
 }
+
+func determineSize(bitCount int) int {
```

Comment:
```suggestion
func subnetCount(bitCount int) int {
```

or `subnetSize` ?

Reply:
`determineSize` was referring to its byte size rather than anything to do with a subnet count.